ktxspec.adoc

```diff
@@ -970,29 +981,41 @@ The ETC1S image format is indicated by `colorModel` `KDF_DF_MODEL_ETC1S`
 DFD must be as described in Section 5.6.11 _KHR_DF_MODEL_ETC1S_ of
 <<KDF13>>. Because ETC1S does not support an alpha component, Basis
 Universal uses 2 _slices_, (_planes_ in DFD-speak).
-This color model provides channel Ids, e.g `KHR_DF_CHANNEL_ETC1S_RGB` that
-must be used to indicate the use of a slice. Whether there
-are 1 or 2 slices depends on the pre-deflation components as detailed
-below along with the channel Ids to use.
+This color model provides the following channel ids that must be used to
+indicate the use of a slice.
 +
-_One-component images_ must have the component value replicated in all 3
-components of the _rgb_ slice of the compressed data and have no _alpha_
-slice. `KHR_DF_CHANNEL_ETC1S_RRR`.
-+
-_Two-component images_ must have the R component replicated in all 3
-components of the rgb slice and the G component replicated in all 3
-components of the alpha slice. `KHR_DF_CHANNEL_ETC1S_RRR`
-+ `KHR_DF_CHANNEL_ETC1S_GGG`
-+
-_Three-component images_ must have their RGB components in the rgb
-slice and no alpha slice. `KHR_DF_CHANNEL_ETC1S_RGB`
+[width=100%,align=center,cols="<30,^10,<60",options=header]
+|===
+| Id | Value | Type 
+|`KHR_DF_CHANNEL_ETC1S_RGB` | 0  | 3 component: opaque color. RGB
+                                   components in the rgb channels.
+|`KHR_DF_CHANNEL_ETC1S_RRR` | 3  | 1 component: R component replicated
+                                   in all 3 rgb channels for better
+                                   compression results.
+|`KHR_DF_CHANNEL_ETC1S_GGG` | 4  | Not used independently. 1 component:
+                                   G component replicated in all 3 rgb
+                                   channels
+|`KHR_DF_CHANNEL_ETC1S_AAA` | 15 | Not used independent;y. 1 component:
+                                   alpha component replicated in all 3
+                                   rgb channels.
+|===
 +
-_Four-component images_ must have their RGB components in the rgb slice
-and the alpha component replicated in all 3 components of the alpha
-slice. `KHR_DF_CHANNEL_ETC1S_RGB` + `KHR_DF_CHANNEL_ETC1S_AAA`
+Whether there are 1 or 2 slices depends on the pre-deflation components
+as detailed in the following table of valid channel id combinations.
 +
-The only valid combinations of more than one slice are `RGB` + `AAA`
-or `RRR` + `GGG` with `RGB` or `RRR` being the first slice.
+[width=100%,align=center,cols="<65,<35",options=header]
+|===
+| Combination | Description
+|`KHR_DF_CHANNEL_ETC1S_RGB` | One slice, opaque color.
+|`KHR_DF_CHANNEL_ETC1S_RGB` + `KHR_DF_CHANNEL_ETC1S_AAA` | Two slices,
+                                                           color + transparency.
```

Comment:
For consistency, consider using "alpha" instead of "transparency" here.

Reply:
Thanks. Done.